### DISCUSSION TITLE
Will not import code from CopperSpice?

### DISCUSSION THREAD

**Discussion by ghost** [RolandHughes/copperspice#13] [General]
Here is your statement on README.md:

```
This is project is not maintained by the CopperSpice project and we no longer accept code from them.
```

Not even fixes in the CMake build scripts?

For example, this is supposed to fix the issue about GObject2 being not found:

https://github.com/copperspice/copperspice/commit/2f6bce3439acd752dd0ec512e3cfdfbf3e688878

**Comment by RolandHughes** [comment]
I'm not pulling anymore from them. It's not physically possible. The branding changes means there is no longer an "upstream."
